doc/onboarding.md

```diff
@@ -169,6 +169,7 @@ Landing a PR
     * `Reviewed-By: human <email>`
       * Easiest to use `git log` then do a search
       * (`/Name` + `enter` (+ `n` as much as you need to) in vim)
+      * Only include collaborators who have commented "LGTM"
```

Comment:
But currently the CTC members & collaborators are different lists in our [README.md#Current Project Team Members](https://github.com/nodejs/node#current-project-team-members), I thought that might be have something confused to new comers :-)